### PR TITLE
fix(zk_toolbox): Set proper artifacts

### PR DESCRIPTION
**Pull request by Deniallugo**
## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.

### CODE REVIEW

File: core/tests/recovery-test/src/index.ts [hunk 1]
```diff
@@ -147,6 +147,43 @@ export class NodeProcess {
...
+        }
+
+        let signalNumber;
+        if (signal == 'KILL') {
+            signalNumber = 9;
+        } else {
+            signalNumber = 15;
+        }
+        try {
+            console.log('Main PID', this.childProcess.pid);
```
Comment (manuelmauro) [inline core/tests/recovery-test/src/index.ts:162]: nit: Is this log still useful?

File: docker-compose.yml [hunk 2]
```diff
@@ -3,6 +3,8 @@ services:
   reth:
     restart: always
     image: "ghcr.io/paradigmxyz/reth:v0.2.0-beta.2"
+    ports:
+      - 127.0.0.1:8545:8545
```
Comment (manuelmauro) [inline docker-compose.yml:7]: This is why we use `localhost` now in CI? Why was this needed?

**Review by manuelmauro — APPROVED** [review]
LGTM! 🥳

File: core/tests/recovery-test/src/index.ts (hunk 1 above)
Comment (Deniallugo) [reply core/tests/recovery-test/src/index.ts:162]: I think no

File: docker-compose.yml (hunk 2 above)
Comment (Deniallugo) [reply docker-compose.yml:7]: Not this, but `network_mode: host` in `zk` container

**Pull request merged**